.github/workflows/ci-alpha.yml

```diff
@@ -49,3 +49,35 @@ jobs:
           buildargs: CUSTOM_REGISTRY_URL,CUSTOM_REGISTRY_TOKEN,WEB_SDK_DIST_TAG,UX_KIT_DIST_TAG
           buildoptions: "--pull"
           tags: "${{ env.WEB_SDK_DIST_TAG }}"
+
+  build_frontend_alpha_without_docker:
+    runs-on: ubuntu-latest
+    defaults:
+      run:
+        working-directory: ./frontend
+    steps:
+      - uses: actions/checkout@v2
+      - name: Configure AWS Credentials
+        uses: aws-actions/configure-aws-credentials@v1
+        with:
+          aws-access-key-id: ${{ secrets.VOXEET_AWS_ACCESS_KEY_ID }}
+          aws-secret-access-key: ${{ secrets.VOXEET_AWS_SECRET_ACCESS_KEY }}
+          aws-region: ${{ secrets.VOXEET_AWS_REGION }}
+      - name: build frontend for develop - alpha Web SDK
+        uses: actions/setup-node@v2
+        with:
+          node-version: "16.9.1"
+      - run: |
+          yarn install
+          yarn build
+      - name: publish frontend for develop - alpha Web SDK
+        uses: jakejarvis/s3-sync-action@master
+        with:
+          args: --acl public-read --follow-symlinks --delete
+        env:
+          AWS_S3_BUCKET: dlb-capi-showcase # this should likely go to secrets
+          AWS_ACCESS_KEY_ID: ${{ secrets.VOXEET_AWS_ACCESS_KEY_ID }}
+          AWS_SECRET_ACCESS_KEY: ${{ secrets.VOXEET_AWS_SECRET_ACCESS_KEY }}
+          AWS_REGION: ${{ secrets.VOXEET_AWS_REGION }}
+          SOURCE_DIR: "./frontend/dist"
```

Comment:
as you have already 
```
defaults:
      run:
        working-directory: ./frontend
```
May be you should have only `dist` here but to be tested